src/BuildingRegistry/Building/Events/BuildingGeometryWasImportedFromGrb.cs

```diff
@@ -0,0 +1,137 @@
+namespace BuildingRegistry.Building.Events
+{
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using Be.Vlaanderen.Basisregisters.EventHandling;
+    using Be.Vlaanderen.Basisregisters.GrAr.Common;
+    using Be.Vlaanderen.Basisregisters.GrAr.Provenance;
+    using Datastructures;
+    using Newtonsoft.Json;
+    using NodaTime;
+
+    [EventTags(EventTag.For.Sync, EventTag.For.Edit, Tag.Building)]
+    [EventName(EventName)]
+    [EventDescription("Het gebouw werd geimporteerd door GRB.")]
+    public sealed class BuildingGeometryWasImportedFromGrb : IBuildingEvent
+    {
+        public const string EventName = "BuildingGeometryWasImportedFromGrb"; // BE CAREFUL CHANGING THIS!!
+
+        [EventPropertyDescription("Objectidentificator van het gebouw.")]
+        public int BuildingPersistentLocalId { get; }
+
+        [EventPropertyDescription("GRB identificator.")]
+        public long Idn { get; }
+
+        [EventPropertyDescription("Versiedatum.")]
+        public Instant VersionDate { get; }
+
+        [EventPropertyDescription("Einddatum (null indien niet van toepassing).")]
+        public Instant? EndDate { get; }
+
+        [EventPropertyDescription("Versie GRB identificator.")]
```

Comment:
review these descriptions, not all have valid values

Reply:
I would wait for Anais to give proper descriptions.